packages/web-app-search/src/portals/SearchBar.vue

```diff
@@ -281,18 +309,43 @@ export default {
     isPreviewElementActive(searchId) {
       const previewElements = this.$refs.optionsDrop.$el.querySelectorAll('.preview')
       return previewElements[this.activePreviewIndex]?.dataset?.searchId === searchId
+    },
+    showSearchBar() {
+      document.getElementById('files-global-search-bar').style.visibility = 'visible'
+      document.getElementsByClassName('oc-search-input')[0].focus()
+      this.showCancelButton = true
+    },
+    cancelSearch() {
+      document.getElementById('files-global-search-bar').style.visibility = 'hidden'
+      this.showCancelButton = false
+    },
+    onResize() {
+      const searchBarEl = document.getElementById('files-global-search-bar')
+      if (window.innerWidth > 639) {
```

Comment:
shouldnt this be possible via css? I mean its fine with me just weird to see this number in code rather than css

Reply:
We need it unfortunately to set the `visible` state back again if you search on a small screen and resize your screen to full again. Without, the search would stay hidden because `showSearchBar()` actively sets `style.visibility = 'visible'`.